src/models/userSeasonStat.ts

```diff
@@ -0,0 +1,110 @@
+import mongoose from 'mongoose';
+import { IUserSeasonStat } from './types/userSeasonStat';
+
+// SCHEMA SETUP
+const userSeasonStatSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'user',
+    required: true,
+  },
+  seasonId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'season',
+    required: true,
+  },
+
+  rating: {
+    type: Number,
+    default: 1500, // TODO-kev: Consider if there is an enum to share here
```

Comment:
nah, maybe have this as a constant somewhere. Like `DEFAULT_STARTING_ELO = 1500` somewhere.